libhb/encavcodec.c

```diff
@@ -180,10 +181,11 @@ int encavcodecInit( hb_work_object_t * w, hb_job_t * job )
         }
     }
 
+    codec = avcodec_find_encoder_by_name(codec_name);
```

Comment:
It's *possible* to get here with codec_name == NULL. It would be a code error if it happened, but possible none the less.  It would be nice to have a test here for codec_name == NULL and an error message indicating that an invalid encoder was specified.  You could just move the default case out of the switch above.

Reply:
How about this solution?